README.md

```diff
@@ -117,12 +139,32 @@ pretrained model for semantic segmentation and run it on a point cloud of our
 dataset. See the [model zoo](#model-zoo) for obtaining the weights of the
 pretrained model.
 ```python
-model = ml3d.models.RandLANet()
-pipeline = ml3d.pipelines.SemanticSegmentation(model=model, dataset=dataset, device="gpu")
-
+import os
+import open3d.ml as _ml3d
+import open3d.ml.torch as ml3d
+
+cfg_file = "ml3d/configs/randlanet_semantickitti.yml"
+cfg = _ml3d.utils.Config.load_from_file(cfg_file)
+
+model = ml3d.models.RandLANet(**cfg.model)
+#path_dataset = cfg.dataset.pop('dataset_path', "/path/to/your/dataset")
+cfg.dataset['dataset_path'] = "/path/to/your/dataset"
+dataset = ml3d.datasets.SemanticKITTI(cfg.dataset.pop('dataset_path', None), **cfg.dataset)
+pipeline = ml3d.pipelines.SemanticSegmentation(model, dataset=dataset, device="gpu", **cfg.pipeline)
+
+# download the weights.
+ckpt_folder = "./logs/"
+os.makedirs(ckpt_folder, exist_ok=True)
+ckpt_path = ckpt_folder + "randlanet_semantickitti_202009090354utc.pth"
+randlanet_url = "https://storage.googleapis.com/open3d-releases/model-zoo/randlanet_semantickitti_202009090354utc.pth"
+if not os.path.exists(ckpt_path):
+    cmd = "wget {} -O {}".format(randlanet_url, ckpt_path)
+    os.system(cmd)
+    
 # load the parameters.
-pipeline.load_ckpt(ckpt_path='randlanet_semantickitti_202009090354utc.pth', is_train=False)
+pipeline.load_ckpt(ckpt_path=ckpt_path)
 
+test_split = dataset.get_split("test")
 data = test_split.get_data(0)
```

Comment:
Can we add a line at the bottom that you can also do all this using our predefined script ?

Reply:
done